index.js

```diff
@@ -1,13 +1,17 @@
 const { exec } = require('child_process');
 const loaderUtils = require('loader-utils');
+const omit = require('lodash.omit');
 
 module.exports = function(source) {
-    var callback = this.async();
-    var options = loaderUtils.getOptions(this);
-    var command = exec(options.script, function(err, result) {
+    const options = loaderUtils.getOptions(this);
+    const callback = this.async();
+    const scriptOptions = omit(options, 'script');
```

Comment:
I like this idea. Could you do this without using lodash? I don't want to have unneeded dependencies. Maybe something like;

```
const { script, ...options } = loaderUtils.getOptions(this);
exec(script, options, function() {
  // 
})
```

I didn't test this, and this project isn't fresh enough in my mind to remember if ES6 features will work here.

Reply:
will give it a try 👍